main.go

```diff
@@ -67,7 +68,9 @@ func main() {
 		"The controller will load its initial configuration from this file. "+
 			"Omit this flag to use the default configuration values. ")
 
-	opts := zap.Options{}
+	opts := zap.Options{
+		TimeEncoder: zapcore.RFC3339NanoTimeEncoder,
```

Comment:
Thanks a lot. Can you add this to the integration tests as well please?
Probably in `Framework.Setup` at `test/integration/framework`

It was driving me crazy when debugging :)